bin/phpcs-changed

```diff
@@ -44,9 +44,34 @@ $fileNamesExpanded = [];
 foreach( $fileNames as $file ) {
 	if ( is_dir($file) ) {
 		$iterator = new RecursiveIteratorIterator(new RecursiveCallbackFilterIterator(new RecursiveDirectoryIterator($file, (RecursiveDirectoryIterator::SKIP_DOTS | FilesystemIterator::FOLLOW_SYMLINKS)), function($file, $key, $iterator){
-			return $iterator->hasChildren() || ( $file->isFile() && '.php' === substr($file->getFilename(),-4) ) ? true : false;
+			$extensions = [
+				'php',
+				'inc',
+				'js',
+				'css',
+			];
+			if ($file->isFile()) {
```

Comment:
Could we replace most of this block with `if (in_array(pathinfo($file->getFilename(), PATHINFO_EXTENSION), $fileExtensions, true)) {`?

Reply:
oh, I see that you copied this logic from phpcs... hm. Well, in that case if you think it's a good idea I'm ok with keeping it, but it's quite hard to read so maybe we can put it into its own function somewhere. At least then it would (eventually) be testable and/or could easily be replaced.

Reply:
Sorry for not making it explicit. Yeah, I copied the logic as the same logic would be used internally by phpcs. Thus there should, ideally, be no differences in there, in order to prevent confusion.

Moving the logic to it's own function sounds like a good plan. Did you have any specific file in mind when saying "somewhere"? :-)

Reply:
> Moving the logic to it's own function sounds like a good plan. Did you have any specific file in mind when saying "somewhere"? :-)

Maybe in https://github.com/sirbrillig/phpcs-changed/blob/master/PhpcsChanged/Cli.php?